utils/update_checkout/update_checkout/update_checkout.py

```diff
@@ -637,6 +645,13 @@ def main():
         default=os.path.join(SCRIPT_DIR, os.pardir,
                              "update-checkout-config.json"),
         help="Configuration file to use")
+    parser.add_argument(
+        "--additional-scheme-config",
+        help="""Configs containing additional branch-schemes to append to the
+        configration. Can be specified multiple times, each will be appended in
+        order.""",
+        action="append",
+        default=[])
```

Comment:
@porglezomp how would we feel about just converting the existing `config` var to be a list which could be applied multiple times. So instead of 
```
update-checkout.py --config update-config.json --additional-scheme-config additional-theme.json
```

We could just unify it to a single variable
```
update-checkout.py --config update-config.json --config additional-theme.json
```

And then loop through them. They appear to be handled the exact same way

Reply:
we could potentially store it in a `configs` var for naming

Reply:
The reason I didn't do it like that is that the first config specifies extra variables like the fetch URLs and the supported repos, but the latter configs only provide the `branch-schemes` key. If we made them all uniform we'd have to decide how we want to handle merging the extra properties.

Some possibilities if we want to follow that route:
- Take the values only from the first config (matching this behavior)
- Validating that the extra configs don't define those
- Checking if the values match

But making them distinguished flags like I did initially makes the asymmetry between the configs more explicit.